operators/dream_texture.py

```diff
@@ -105,7 +105,9 @@ def done_callback(future):
             result: ImageGenerationResult = future.result(last_only=True)
             for i, result_image in enumerate(result.images):
                 seed = result.seeds[i]
-                image = bpy_image(str(seed), result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
+                prompt_string = bpy.data.scenes["Scene"].dream_textures_prompt.prompt_structure_token_subject
+
+                image = bpy_image(str(prompt_string + " " + str(seed)), result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```

Comment:
I think an f-string would be a bit simpler here. And putting the seed in parentheses or after a dash may be easier to read:

```suggestion
                image = bpy_image(f"{prompt_string} ({seed})", result_image.shape[1], result_image.shape[0], result_image.ravel(), last_data_block)
```